### PR TITLE
add prefix path

### LINKED ISSUE #86: Add provision to custom path prefix #hactoberfest #HACTOBERFEST2022
### Is there an existing feature request for this?

- [X] I have searched the existing issues

### Summary

To update custom prefix path

### Why should this be worked on?

On setting this path the user can udpdate their own  prefix in ingress.

### PR DESCRIPTION
**Pull request by sreekesh93**
Signed-off-by: sreekesh93 <sreekeshsbkv@gmail.com>

# Pull Request Template

## Description

Please include a summary of the change. Please also include relevant motivation and context. List any dependencies that are required for this change.

Fixes #86 

## Type of change

Please delete options that are not relevant.

- [x] Bug fix (non-breaking change which fixes an issue).

## How Has This Been Tested?
started keploy project from local and validated UI path prefix

### CODE REVIEW

**Comment by sreekesh93** [comment]
please add #hactoberfest tag to the label thanks

**Comment by sreekesh93** [comment]
@nehagup please let me know if I missed anything. If not can you please review this PR

**Comment by nehagup** [comment]
@sreekesh93 Yes I will review it.

File: gatsby-config.js [hunk 1]
```diff
@@ -1,4 +1,5 @@
 module.exports = {
+  pathPrefix: process.env.PATH_PREFIX || "/",
```
Comment (slayerjain) [inline gatsby-config.js:2]: please name this something like `KEPLOY_PATH_PREFIX` because I got confused with `PREFIX_PATHS` gatsby inbuilt variable

**Comment by sreekesh93** [comment]
@slayerjain do you see any concerns with changes still?

**Comment by sreekesh93** [comment]
@slayerjain any updates on this PR?

**Comment by slayerjain** [comment]
Hi @sreekesh93,  I'm sorry for the late reply. You need to update the `PATH_PREFIX` variable here too.

**Comment by sreekesh93** [comment]
@slayerjain thanks updated the same, missed to push last time :)

**Pull request merged**